tfhe/web_wasm_parallel_tests/Makefile

```diff
@@ -23,3 +23,13 @@ bench: run_server
 
 .PHONY: bench-ci # Run benchmarks on web client in CI
 bench-ci: run_server
+
+.PHONY: fmt # Format Javascript code
+fmt:
```

Comment:
needs npm installed ? don't know if it's already installed in here

Reply:
At this point we consider `npm` has being installed. Look at the `run_server` recipe in this file.

Reply:
ok thx